db/changes.go

```diff
@@ -281,6 +436,17 @@ func makeChangeEntry(logEntry *LogEntry, seqID SequenceID, channelName string) C
 	return change
 }
 
+func makeRevocationChangeEntry(logEntry *LogEntry, seqID SequenceID, channelName string, idOnly bool) ChangeEntry {
```

Comment:
Does 'omitRevID' make more sense than idOnly, given that we're sending more than just id?  In particular I wasn't sure at first whether we still send seqID.seq in the case where we're omitting rev.

Reply:
Yeah thats makes more sense.